getting_started/step_by_step/scripting_first_script.rst

```diff
@@ -231,6 +278,14 @@ At the bottom of the script, define the function:
         Rotation += _angularSpeed * (float)delta;
     }
 
+ .. code-tab:: cpp C++
+
+    void _process(double p_delta) override {
+        // Note that rotation is a private member variable of Node2D, so the set_rotation function must be used.
+        // You'll find that a lot of member variables that are public in gdscript/C# are instead private in GDExtension C++.
```

Comment:
It isn't so much a "private member" as properties in godot-cpp must be accessed via setters and getters.

```suggestion
        // Note that properties (like rotation) are accessed via setters and getters in godot-cpp.
GDExtension C++.
```